conf/solr/schema.xml

```diff
@@ -351,6 +351,7 @@
     <field name="productionPlace" type="text_en" multiValued="true" stored="true" indexed="true"/>
```

Comment:
I'll just leave this comment here on the first file that is changed...

Can we please get a release note?